dts/arm/nuvoton/npcx7m6fb.dtsi

```diff
@@ -21,6 +21,12 @@
 		reg = <0x200C0000 DT_SIZE_K(62)>;
 	};
 
+	/* RAM space used by Booter */
+	bt_ram: memory@200cf800 {
```

Comment:
Suggest naming this node "bootloader_ram".  The acronym "bt" is widely used by the Bluetooth code in Zephyr.

Reply:
done